rewrite-maven/src/main/java/org/openrewrite/maven/search/FindProperties.java

```diff
@@ -36,9 +36,7 @@
 @Value
 public class FindProperties extends Recipe {
 
-    @Option(displayName = "Property pattern",
-            description = "Regular expression pattern used to match property tag names.",
-            example = "guava*")
+    @Option(displayName = "Property pattern", description = "Regular expression pattern used to match property tag names.", example = "guava*")
```

Comment:
I know this isn't part of your changes, but I find the description example value here a little confusing.
- We say the pattern is a regular expression, yet below we replace `.` with `\\.` and replace `*` with `.*`. Would that not make this a glob pattern instead?
- If the value _is_ a regular expression, then `guava*` doesn't make much sense; that'd match `guav`, `guava` & `guavaaaa` for example, were it not for the above special handling.
- How does the algorithm below handle proper regular expression patterns like `\\.` or `.*`? it seems we'd duplicate the replacements, which is at best confusing, at worst wrong.

Perhaps we should reevaluate how `propertyPattern` is used, and adjust our description here accordingly.